app/views/layouts/application.html.erb

```diff
@@ -24,6 +24,11 @@
       <% end %>
       <%= yield %>
     </div>
+    <script>
```

Comment:
Please remove this inline javascript.

Reply:
Okay. I was just testing, this wasn't suppose to be here